src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -596,9 +596,18 @@ def reload_module(module):
 
 def get_default_admin_username():
     try:
-        return getpass.getuser()
+        username = getpass.getuser()
     except KeyError:
-        return None
+        username = None
+    disallowed_user_names = [
```

Comment:
How about some descriptions about this list?

Reply:
comments

Reply:
OK

Reply:
hi @qwordy where the list from? In this case, what's impact of failure? with clear message?

Reply:
> 
> 
> hi @qwordy where the list from? In this case, what's impact of failure? with clear message?

Copied from code written by Yugang Wang back to 2017